groups/templates/groups/group_detail_base.html

```diff
@@ -22,6 +22,7 @@
                 <li><a href="{{ discussion.get_absolute_url }}">{{ discussion.name }}</a></li>
             </ul>
         {% endfor %}
+        </li>
```

Comment:
Looking at the whole file, this `</li>` and its matching opening tag want to be inside the `{% for %}` tags:

```
<ul>
    {% for blah in blah %}
        <li>
            <ul> ... </ul>
        </li>
    {% endfor %}
</ul>